web/client-api/src/main/java/io/deephaven/web/client/api/grpc/UnaryWithHeaders.java

```diff
@@ -5,39 +5,40 @@
 
 import elemental2.promise.IThenable;
 import elemental2.promise.Promise;
+import io.deephaven.javascript.proto.dhinternal.browserheaders.BrowserHeaders;
 import io.deephaven.javascript.proto.dhinternal.grpcweb.Grpc;
+import io.deephaven.javascript.proto.dhinternal.grpcweb.client.RpcOptions;
 import io.deephaven.javascript.proto.dhinternal.grpcweb.grpc.Code;
 import io.deephaven.javascript.proto.dhinternal.grpcweb.unary.UnaryOutput;
 import io.deephaven.javascript.proto.dhinternal.grpcweb.unary.UnaryRpcOptions;
-import io.deephaven.web.client.api.WorkerConnection;
 
 public class UnaryWithHeaders {
 
     /**
      * Improbable-eng's grpc-web implementation doesn't pass headers to api callers - this changes the contract a bit so
      * that we can get a typed UnaryOutput with the headers/trailers intact.
      * 
-     * @param connection provides access to the metadata and the server url
      * @param methodDescriptor the service method to invoke
      * @return a promise that will resolve to the response plus headers/trailers, or reject with the headers/trailers
      * @param <Res> type of the message object
      */
-    public static <Req, Res> Promise<UnaryOutput<Res>> call(WorkerConnection connection, Object methodDescriptor,
+    public static <Req, Res> Promise<UnaryOutput<Res>> call(String host, BrowserHeaders metadata, RpcOptions options,
```

Comment:
I think the param js docs are out of sync here.

Reply:
javadoc ;) but yes, thank you